docs/configuration/overview/tracing.rst

```diff
@@ -12,31 +12,30 @@ the only one supported.
 
   {
     "http": {
-      "sinks": []
+      "driver": "{...}"
     }
   }
 
 http
   *(optional, object)* Provides configuration for the HTTP tracer.
 
-sinks
-  *(optional, string)* Provides list of sinks traces are sent to.
+driver
+  *(optional, object)* Provides the driver that traces are sent to.
```

Comment:
Something along these lines: provides the driver that handles trace and span creation.